lib/ansible/modules/package_facts.py

```diff
@@ -384,7 +386,7 @@ def main():
 
     # start work
     global module
-    module = AnsibleModule(argument_spec=dict(manager={'type': 'list', 'default': ['auto']},
+    module = AnsibleModule(argument_spec=dict(manager={'type': 'list', 'elements': 'str', 'default': ['auto']},
```

Comment:
```suggestion
    module = AnsibleModule(argument_spec=dict(manager={'type': 'list', 'elements': 'str', 'default': ['auto'], choices=['apt', 'auto', 'pacman', 'pkg', 'portage', 'rpm']},
```

Reply:
I'm not sure If I've changed correctly as your suggestion, could you please review this?
https://app.shippable.com/github/ansible/ansible/runs/169575/2/tests

Reply:
Also this:
https://app.shippable.com/github/ansible/ansible/runs/169575/1/tests

Reply:
```suggestion
    module = AnsibleModule(argument_spec=dict(manager={'type': 'list', 'elements': 'str', 'default': ['auto'], choices: ['apt', 'auto', 'pacman', 'pkg', 'portage', 'rpm']},
```

Reply:
Oops. Changed it.